dashboards/grafana-dashboard-kessel-spicedb.configmap.yaml

```diff
@@ -0,0 +1,4702 @@
+              "selected": false,
+              "text": "authzed.api.v1.SchemaService",
+              "value": "authzed.api.v1.SchemaService"
+            },
+            "datasource": {
+              "type": "prometheus",
+              "uid": "$datasource"
+            },
+            "definition": "label_values(grpc_service)",
+            "hide": 0,
+            "includeAll": false,
+            "multi": false,
+            "name": "service",
+            "options": [],
+            "query": {
+              "qryType": 1,
+              "query": "label_values(grpc_service)",
+              "refId": "PrometheusVariableQueryEditor-VariableQuery"
+            },
+            "refresh": 1,
+            "regex": "",
+            "skipUrlSync": false,
+            "sort": 0,
+            "type": "query"
+          },
+          {
+            "current": {
+              "selected": false,
+              "text": "crcs02ue1-prometheus",
+              "value": "fdy3k2jvla39cc"
+            },
+            "hide": 0,
+            "includeAll": false,
+            "label": "Datasource",
+            "multi": false,
+            "name": "datasource",
+            "options": [],
+            "query": "prometheus",
+            "queryValue": "",
+            "refresh": 1,
+            "regex": "/(crcp01ue1-prometheus)|(crcs02ue1-prometheus)/",
+            "skipUrlSync": false,
+            "type": "datasource"
+          },
+          {
+            "current": {
+              "selected": false,
+              "text": "ReadRelationships",
+              "value": "ReadRelationships"
+            },
+            "datasource": {
+              "type": "prometheus",
+              "uid": "${datasource}"
+            },
+            "definition": "label_values(grpc_method)",
+            "description": "gRPC Method",
+            "hide": 0,
+            "includeAll": true,
+            "label": "Method",
+            "multi": false,
+            "name": "method",
+            "options": [],
+            "query": {
+              "qryType": 1,
+              "query": "label_values(grpc_method)",
+              "refId": "PrometheusVariableQueryEditor-VariableQuery"
+            },
+            "refresh": 1,
+            "regex": "",
+            "skipUrlSync": false,
+            "sort": 0,
+            "type": "query"
+          }
+        ]
+      },
+      "time": {
+        "from": "now-1h",
+        "to": "now"
+      },
+      "timepicker": {
+        "hidden": false,
+        "refresh_intervals": [
+          "10s",
+          "15s",
+          "30s",
+          "1m",
+          "5m",
+          "10m",
+          "15m"
+        ],
+        "time_options": [
+          "5m",
+          "15m",
+          "1h",
+          "6h",
+          "12h",
+          "24h",
+          "2d",
+          "7d",
+          "30d"
+        ]
+      },
+      "timezone": "",
+      "title": "Kessel - SpiceDB ",
+      "uid": "gn38yKZnk-import",
+      "version": 19,
```

Comment:
version 1 may be?

Reply:
Ahh good call, thats what i get for copy/paste!